themes/wilson_theme_starterkit/templates/content/node--next-card.html.twig

```diff
@@ -0,0 +1,95 @@
+{#
+/**
+ * @file
+ * Theme override to display a teaser node.
```

Comment:
Comment could be updated here to reference next card node

Reply:
Good catch. I've updated.